docs/contribute/contribute.adoc

```diff
@@ -4,126 +4,169 @@ This document is meant for new OpenSCAP contributors and it describes how to
+should go to the `maint-1.2` branch doesn't have to be true. It's a good practice
 to investigate the correct branch first or ask experienced developers on our
 FreeNode IRC channel called `#openscap` or
 link:https://www.redhat.com/mailman/listinfo/open-scap-list[mailing list].
 
-Once you have forked the repository and decided what branch the fix will go into
-you can clone your forked repository.
+NOTE: The default branch of the openscap repository is set to the `maint-1.2`.
+
+First you need to clone your forked repository:
 [[app-listing]]
 [source,bash]
 ----
 $ git clone git@github.com:adam/openscap.git
 ----
 
-
-Now you can create a new branch, which we will call _fix_455_, where you can work on the fix. Remember that
-the name of the new branch will appear in the commit message when your fix is
-merged so choose the name wisely.
+Once you have forked the repository and decided that the fix will go into the
+`maint-1.2` branch you can create a new branch from it, which we will call
+_fix_455_, where you can work on the fix. Remember that the name of the new
+branch will appear in the commit message when your fix is merged so choose the
+name wisely:
 
 [[app-listing]]
 [source,bash]
 ----
-# checkout to the maint-1.0 branch
-$ git checkout maint-1.0
+$ cd openscap/
 # create a new branch
 $ git checkout -b fix_455
 ----
 
-NOTE: The default branch of the openscap repository is set to the `maint-1.2`.
-Our fix will go into the `maint-1.0` so we have to switch to this branch before
-creating a new one.
+NOTE: In the code snippets, lines starting with `#` are comments and lines
+starting with `$` are commands.
+
+On the other hand, if you decided that the fix will go into the `master` branch
+you have to switch to this branch first before creating a new one:
+[[app-listing]]
+[source,bash]
+----
+# create a new local branch to track the remote master branch
+$ git checkout -b master remotes/origin/master
+# and now create a new branch from the master branch which will contain your fix
+$ git checkout -b fix_455
+----
+
 
 == Fix the issue
-NOTE: OpenSCAP is licensed under LGPLv2. Any fixes or improvements must be licensed
-under the same license to be included. Check the COPYING file in the repository
-for more details.
+NOTE: OpenSCAP is licensed under _LGPL v2.1_. Any fixes or improvements must be
+licensed under the same license to be included. Check the COPYING file in the
+repository for more details.
 
 Now you can work on the fix. Try to create small commits which can be easily
 reviewed and make self-explaining commit messages. The
 link:http://chris.beams.io/posts/git-commit/[How to Write a Git Commit
 Message] article could help you with that. Nobody will review a pull request
 which contains a four thousand new lines of code.
 
-NOTE: Since we're fixing an issue number 455 make sure that your commit message
-contain a
+NOTE: Since you're fixing the issue number 455 make sure that your commit
```

Comment:
drop `the`. I am not sure why but I am pretty sure it's either `an issue`, `the issue that blablabla` or `issue number XYZ`.

Reply:
I don't think it can be `an issue` as in this case we are referring to the specific issue with number 455, but if you are sure, I will remove `the`.